docroot/sites/default/files/config_SKRbKjrsGZbCwa_q0wg8DYZpUGb3pdwwxawoq_xE0FXjABmFBcdqfoyLjvWYMn74C7COWTFr6w/sync/seckit.settings.yml

```diff
@@ -28,13 +34,23 @@ seckit_clickjacking:
   x_frame: '1'
   x_frame_allow_from: ''
 seckit_ssl:
-  hsts: false
+  hsts: true
   hsts_subdomains: false
-  hsts_max_age: 1000
+  hsts_max_age: 2592000
   hsts_preload: false
 seckit_various:
   from_origin: false
   from_origin_destination: same
   disable_autocomplete: false
+  referrer_policy: true
+  referrer_policy_policy: no-referrer-when-downgrade
 _core:
   default_config_hash: 1QSPl6Lx4K-hAcSOsRUFyntOnghLfpsDzjY6tbhnAVY
+seckit_ct:
+  expect_ct: true
+  max_age: 0
+  report_uri: ''
+  enforce: false
```

Comment:
With enforcement disabled, this would be in "report only" mode, which means it would need a `report_uri` value to send violations to (without actually preventing the browser from displaying the page). I'm not sure if Drupal 8 has such an endpoint but as it stands this config would not be not enforcing or reporting cert transparency violations, i.e. it's not really doing anything (which may be desired, but if so, `expect_ct` can probably be `false` I imagine).